app/views/onboardings/_task_card.html.erb

```diff
@@ -10,6 +10,11 @@
     localStorage.setItem('task-card-closed', 'yes');
   }
 </script>
+<style>
```

Comment:
Should we include a comment here so we don't accidentally remove this going forward?

Reply:
Yeah this couldn't hurt. But of note: We actually _can_ remove this after not too long. The comment should probably give all the context. I should have included that.

But a week from now we should safely feel like we can remove this.

This is the PR meant to deal with this but I sort of let it fall off... I wasn't totally sure I liked the solution but it would be good to revisit this shortly...

https://github.com/thepracticaldev/dev.to/pull/6301

Reply:
Ah, gotcha. If it's not long term, I feel like we don't need to bother  🙃